blivet/devices/__init__.py

```diff
@@ -19,7 +19,7 @@
 # Red Hat Author(s): David Lehman <dlehman@redhat.com>
 #
 
-from .lib import device_path_to_name, device_name_to_disk_by_path, ParentList
+from .lib import device_path_to_name, device_name_to_disk_by_path, ParentList, Tags
```

Comment:
Is this a leftover from something or intentional change to make ``Tags`` importable from ``devices``?